pkgs/development/python-modules/django-maintenance-mode/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchPypi, buildPythonPackage,
+  mock, django }:
+buildPythonPackage rec {
+  pname = "django-maintenance-mode";
+  version = "0.13.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "16yfw85r6my14vbrf9g2hj5p7ivgjvrd16s9prsjzqj9k8qikch2";
+  };
+
+  doCheck = false;
```

Comment:
Could you add a comment to explain why tests are disabled? (probably because setting up a django environment for nix-build can be problematic)

Reply:
Sure!